### DISCUSSION TITLE
Unterstützung MI-1200 Wechselrichter?

### DISCUSSION THREAD

**Discussion by sk-seo** [tbnobody/OpenDTU#1391] [Q&A] [answered]
Hallo zusammen,
ich betreibe eine PV-Anlage, in der zwei MI-1200 und ein HM-1200 werkeln.
Nach den dese Woche von Heise.de berichteten Sicherheitsproblemen überlege ich, auf OpenDTU zu wechseln.
Nur werden die beiden MI-1200 anscheinend nicht unterstützt. Besteht eine Möglichkeit, diese ebenfalls zu unterstützen bzw. woran liegt es, dass die MI-1200 nicht mit OpenDTU kompatibel sind. Meine jetzige Hoymiles DTU kommt ja auch mit beiden zurecht?
VG

**Comment by Omega13x** [comment, marked as answer]
Du kannst es ja mal mit Ahoy-DTU versuchen. Die unterstützen zwar nicht expliziet den Mi-1200, aber den 1500er und auch kleinere. [https://github.com/lumapu/ahoy/blob/main/User_Manual.md](url). Du kannst es ja mal versuchen. Die Bastelausführung kostet so zw. 15 und 25€. Hole dir aber auf jeden Fall einen ESP32 Wroom mit 38 Pins. Der ESP8266 kommt mittlerweile an seine Grenzen. Hier ein Beispiel.: [https://www.az-delivery.de/products/esp32-developmentboard](url). Dort bekommst du auch den Rest. Als Netzteil empfehle ich ein gutes Handy-Ladegerät. besser aber ein Raspberry-Pi Netzteil. Schaue auch mal auf Discord rein. Da gibt es jede Menge input. [https://discord.gg/WzhxEY62mB](url)

**Comment by sk-seo** [comment]
Hi Omega,
vielen Dank! Seh' ich mir an.
